src/taskEvents/utils.js

```diff
@@ -107,6 +107,99 @@ async function getData(data) {
   };
 }
 
+async function getUserSkills(id) {
+  try {
+    const res = await fetch(`${BASE_URL}/users/${id}/skills`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+    const userSkills = await res.json();
+    return userSkills;
+  } catch (error) {
+    alert(error);
+  }
+}
+
+async function getTagLevelOptions() {
+  try {
+    const levelsResponse = await fetch(`${BASE_URL}/levels`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+    const tagsResponse = await fetch(`${BASE_URL}/tags`, {
+      method: 'GET',
+      credentials: 'include',
+      headers: {
+        'Content-type': 'application/json',
+      },
+    });
+
+    const { allTags } = await tagsResponse.json();
+    let { allLevels } = await levelsResponse.json();
```

Comment:
Please handle the case when the backend responds with a 500 server error

Reply:
okay